docs/operations/rolling-updates.md

```diff
@@ -27,15 +27,15 @@ For rolling Apache Druid cluster updates with no downtime, we recommend updating
 following order:
 
 1. Historical
-2. \*Overlord (if any)
-3. \*Middle Manager/Indexers (if any)
-4. Standalone Real-time (if any)
-5. Broker
+2. Middle Manager and Indexers (if any)
+3. Broker
+4. Routers
+5. Overlord (Note that you can upgrade the Overlord before any MiddleManager processes if you use [autoscaling-based replacement](#autoscaling-based-replacement).)
```

Comment:
Not sure which one reads better (singular/plural form consistency):
```suggestion
3. Brokers
4. Routers
5. Overlords (Note that you can upgrade the Overlords before any MiddleManagers if you use [autoscaling-based replacement](#autoscaling-based-replacement).)
```

Reply:
Singular since the intro sentence uses "processes"